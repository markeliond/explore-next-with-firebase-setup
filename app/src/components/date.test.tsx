import React from 'react';
import renderer from 'react-test-renderer';

import Date from './date';

test('renders correctly', () => {
    const dateString = '2021-10-21';

    const component = renderer.create(
        <Date dateString={dateString} />
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})