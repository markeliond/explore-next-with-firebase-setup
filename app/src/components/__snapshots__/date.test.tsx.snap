// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<time
  dateTime="2021-10-21"
>
  October 21, 2021
</time>
`;
